amplitude/decayTopology.cc

```diff
@@ -995,8 +996,8 @@ decayTopology::buildInternalData()
 
 interactionVertexPtr
 decayTopology::cloneNode(const nodeDesc& nd,
-                         const bool      cloneInParticles,
-                         const bool      cloneOutParticles)
+                         const bool,
+                         const bool)
 {
```

Comment:
should probably not be removed here, but the parameters should be added to the call to cloneNode in line 1004 (old revision).